exe/cardmaster

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby --disable-gems
```

Comment:
on linux ```env``` doesn't accept arguments
https://www.in-ulm.de/~mascheck/various/shebang/#splitting